src/test/java/de/tum/in/www1/artemis/architecture/ArchitectureTest.java

```diff
@@ -264,8 +263,7 @@ private DescribedPredicate<JavaClass> haveACollectionField() {
 
             @Override
             public boolean test(JavaClass javaClass) {
-                return javaClass.getAllFields().stream().map(field -> field.getRawType().reflect())
-                        .anyMatch(field -> field.equals(List.class) || field.equals(Set.class) || field.equals(Map.class) || field.equals(Collection.class));
+                return javaClass.getAllFields().stream().map(field -> field.getRawType().reflect()).anyMatch(Collection.class::isAssignableFrom);
```

Comment:
`Map'  is not a subclass of `Collection`
```suggestion
                return javaClass.getAllFields().stream().map(field -> field.getRawType().reflect()).anyMatch(type -> Collection.class.isAssignableFrom(type) || Map.class.isAssignableFrom(type));
```